spec/dummy/config/initializers/devise.rb

```diff
@@ -3,327 +3,24 @@
-  # If you want to use other strategies, that are not supported by Devise, or
-  # change the failure app, you can configure them inside the config.warden block.
-  #
-  # config.warden do |manager|
-  #   manager.intercept_401 = false
-  #   manager.default_strategies(scope: :user).unshift :some_external_strategy
-  # end
-
-  # ==> Mountable engine configurations
-  # When using Devise inside an engine, let's call it `MyEngine`, and this engine
-  # is mountable, there are some extra configurations to be taken into account.
-  # The following options are available, assuming the engine is mounted as:
-  #
-  #     mount MyEngine, at: '/my_engine'
-  #
-  # The router that invoked `devise_for`, in the example above, would be:
-  # config.router_name = :my_engine
-  #
-  # When using OmniAuth, Devise cannot automatically set OmniAuth path,
-  # so you need to do it manually. For the users scope, it would be:
-  # config.omniauth_path_prefix = '/my_engine/users/auth'
-
-  # ==> Turbolinks configuration
-  # If your app is using Turbolinks, Turbolinks::Controller needs to be included to make redirection work correctly:
-  #
-  # ActiveSupport.on_load(:devise_failure_app) do
-  #   include Turbolinks::Controller
-  # end
-
-  # ==> Configuration for :registerable
-
-  # When set to false, does not sign a user in automatically after their password is
-  # changed. Defaults to true, so a user is signed in automatically after changing a password.
-  # config.sign_in_after_change_password = true
-
-  # ==> Devise TwoFactor Extension
-  # Configure OTP extension for devise
-
-  # OTP is mandatory, users are going to be asked to
-  # enroll OTP the next time they sign in, before they can successfully complete the session establishment.
-  # This is the global value, can also be set on each user.
-  #config.otp_mandatory = false
-
-  # Drift: a window which provides allowance for drift between a user's token device clock
-  # (and therefore their OTP tokens) and the authentication server's clock.
-  # Expressed in minutes centered at the current time. (Note: it's a number, *NOT* 3.minutes )
-  #config.otp_drift_window = 3
-
-  # Users that have logged in longer than this time ago, are going to be asked their password
-  # (and an OTP challenge, if enabled) before they can see or change their otp informations.
-  #config.otp_credentials_refresh = 15.minutes
-
-  # Users are given a list of one-time recovery tokens, for emergency access
-  # set to false to disable giving recovery tokens.
-  #config.otp_recovery_tokens = 10
-
-  # The user is allowed to set his browser as "trusted", no more OTP challenges will be
-  # asked for that browser, for a limited time.
-  # set to false to disable setting the browser as trusted
-  #config.otp_trust_persistence = 1.month
-
-  # The name of the token issuer, to be added to the provisioning
-  # url. Display will vary based on token application. (defaults to the Rails application class)
   config.otp_issuer = 'dummy'
```

Comment:
When I requested you not add the blanket config comments, I didn't really mean delete ALL the comments :p